src/common/cli_wrapper.cpp

```diff
@@ -145,6 +151,12 @@ void CLIWrapper::parseAliases() {
       bool expand = false;
       if(config_[alias.key].IsSequence()) {
         auto aliasOpts = config_[alias.key].as<std::vector<std::string>>();
+        // Abort if an alias option has an unknown value, i.e. value that has not been defined
+        // in common/aliases.cpp
+        for(const auto &aliasOpt : aliasOpts)
+          if(allowedAliasValues[alias.key].count(aliasOpt) == 0)
```

Comment:
I personally prefer to use 
`if(allowedAliasValues.find(alias.key) == allowedAliasValues::end())`

Reply:
It would be `if(allowedAliasValues[alias.key].find(aliasOpt) == allowedAliasValues[alias.key].end())` and I kept the previous version because it was shorter.